process/stakeholder.md

```diff
@@ -0,0 +1,16 @@
+
+# Stakeholders 
+
+The following chart is the current Stakeholders of OpenSSF along with thier expected engagement, influence and the requirements they have. 
+
+This analysis is useful to understand the various needs of the community.
+
+| Stakeholder       | Description           | Engagement  | Influence | Requirement |
+| :---------------- | :-------------------- | :-----------|:----------|:----------
+| Members   |   This refers to the companies that work with OpenSSF. For an updated list see the [OpenSSF Landscape](https://landscape.openssf.org/).| Employees of member companies can contribute to all areas at their own desired level. |  Various  |   Needs a fully secure supply chain.  |
+| Other Open Source Foundations/Standards Bodies |  This refers the large scale groups and organizations not affilated directly with the OpenSSF. | Various | Various | Needs collaboration on areas of shared interest and in the furtherance of the open source ecosystem.|
+| Governing Board  |   This refers the people in the Board as defined in the [OpenSSF Charter](https://openssf.org/about/charter/).  | <ul><li>Attend each board meeting or send representative in the case of absence.</li><li>As needed sub-committee meetings.</li><li>Pre-work and outside work at a minimum of 1 hours a month.</li></ul> | Overall management of the OpenSSF | Needs clear data and updates on foundation activities to make business decisions. |
+| TAC Members |  This refers the elected members of the TAC as defined in the [OpenSSF Charter](https://openssf.org/about/charter/). | <ul><li>Attend each TAC meeting. </li><li>As needed sub-committee meetings.</li><li>Pre-work and outside work at a minimum of 4 hours a month.</li></ul> |  Reports on the technical initiatives from sub-groups as defined in the [organizational structure](../organizational-structure-overview.md) | Needs clear data and updates from each sub-group. |
+| Open Source maintainers/developers/projects |  This refers the individuals who are working upstream of others in development.  | Various | Various | Need useful tools, processes, and training to do their jobs securely, with minimal friction and loss of velocity. |
+| Open Source Consumers |  This refers the end user. | Participate in educational opportunities. | Particpate in focus groups and surveys as needed. | Need tools and signals to understand and evaluate the security qualities of the software they are using and the supply chains it is created in. |
+
```

Comment:
A potential additional group of stakeholders:
- security researchers: need processes and tools for handling vulnerability disclosures in collaboration with open source projects (e.g., disclosure processes, GitHub/GitLab support)